src/playlist.js

```diff
@@ -0,0 +1,66 @@
+var playItem = require('./playitem.js');
+var setupAutoadvance = require('./autoadvance.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var autoadvanceTimeout = null;
+
+  var playlist = function playlist(list) {
+    if (typeof list === 'number') {
+      return player.playlist.item(list);
```

Comment:
But *we* aren't modyfing the player object. We're calling `videojs.plugin`.
And yes, that can result in conflicts, but I definitely don't want to be adding properties directly onto the player.
Personally, I'm more than happy with the `.item()` method call.